REV_CAPI.cpp

```diff
@@ -23,12 +23,14 @@ OVR_PUBLIC_FUNCTION(ovrResult) ovr_Initialize(const ovrInitParams* params)
 {
 	MH_QueueDisableHook(LoadLibraryW);
 	MH_QueueDisableHook(OpenEventW);
+	MH_QueueDisableHook(FreeLibrary);
 	MH_ApplyQueued();
 
 	g_VRSystem = vr::VR_Init(&g_InitError, vr::VRApplication_Scene);
 
 	MH_QueueEnableHook(LoadLibraryW);
 	MH_QueueEnableHook(OpenEventW);
+	MH_QueueEnableHook(FreeLibrary);
```

Comment:
It's good to see you remembered it, but it's not actually necessary to disable this hook.